### PR TITLE
[TA] Target swagger and service v3.1

### LINKED ISSUE #19529: HealthcarEntity Category needs to be an enum
Need to ask service team to add values to swagger

### PR DESCRIPTION
**Pull request by maririos**
- Updates TA to new stable swagger
- Fixes https://github.com/Azure/azure-sdk-for-net/issues/19529 by exposing `HealthcareEntityCategory`
- TaskName is hidden for now. I will expose it in a future PR. Issue https://github.com/Azure/azure-sdk-for-net/issues/21804
- Targets v3.1 so re-recordings are necessary.
- Various features are not working properly so tests are disabled. All have issues in GitHub and the service has been notified.
- Changed CI to target static endpoint and disabled all AAD tests. The issue to undo this: https://github.com/Azure/azure-sdk-for-net/issues/21782

### CODE REVIEW

**Comment by maririos** [comment]
sdk/textanalytics/tests.yml

File: eng/pipelines/templates/jobs/live.tests.yml [hunk 1]
```diff
@@ -87,10 +90,11 @@ jobs:
...
-        parameters:
-          SubscriptionConfiguration: ${{ parameters.CloudConfig.SubscriptionConfiguration }}
-          SubscriptionConfigurations: ${{ parameters.CloudConfig.SubscriptionConfigurations }}
+      - ${{ if ne(parameters.DisableAzureResourceCreation, 'true') }}:
+        - template: /eng/common/TestResources/build-test-resource-config.yml
+          parameters:
+            SubscriptionConfiguration: ${{ parameters.CloudConfig.SubscriptionConfiguration }}
+            SubscriptionConfigurations: ${{ parameters.CloudConfig.SubscriptionConfigurations }}
 
       - template: /eng/common/TestResources/deploy-test-resources.yml
```
Comment (benbp) [inline eng/pipelines/templates/jobs/live.tests.yml:99]: You'll need to indent this block as well to line up under `DisableAzureResourceCreation`.

**Pull request closed without merging**